### PR TITLE
 [#13064] Use AccountRequestUpdateRequest as parameter  

### LINKED ISSUE #13064: Use AccountRequestUpdateRequest as parameter
**Environment**: `master`

**Description of feature/enhancement**

Currently the AccountRequestUpdate parameters are passed in as separate parameters (see this [link](https://github.com/TEAMMATES/teammates/blob/4682dffb71e4566ed72a9e8a7b2b814e8d16f869/src/web/services/account.service.ts#L121-L136)).

We want to change it to take in one type as the parameter, instead of passing every parameters in separately. An example of how a type is passed in to represent all parameters: [link](https://github.com/TEAMMATES/teammates/blob/4682dffb71e4566ed72a9e8a7b2b814e8d16f869/src/web/services/course.service.ts#L139-L142)

**Justification**

This refactors the code in a more maintainable way, as developers do not need to update the function parameters every time there is a change. Instead, they can just edit the typescript type.

### PR DESCRIPTION
**Pull request by Stain19**
Fixes #13064

### CODE REVIEW

**Comment by ziqing26** [comment]
@Stain19 Hi please fix the linting test as per github action.Thanks!

**Comment by Stain19** [comment]
Changes Made

File: src/web/services/account.service.ts [hunk 1]
```diff
@@ -118,20 +118,11 @@ export class AccountService {
   /**
    * Edits an account request by calling API.
    */
-  editAccountRequest(id: string, name: string, email: string, institute: string,
-    status: AccountRequestStatus, comments: string)
+  editAccountRequest(id: string, accountReqUpdateRequest: AccountRequestUpdateRequest)
   : Observable<AccountRequest> {
```
Comment (jayasting98) [inline src/web/services/account.service.ts:122]: This is just a nit. Would it be possible to combine this into one line? So like

```typescript
editAccountRequest(id: string, accountReqUpdateRequest: AccountRequestUpdateRequest) : Observable<AccountRequest> {
```

If I'm not wrong, the limit is 120, and this single line should only be 117, right?

**Review by jayasting98 — COMMENTED** [review]
Just a nit.

File: src/web/services/account.service.ts (hunk 1 above)
Comment (Stain19) [reply src/web/services/account.service.ts]: Changes Made!!
Apparently nothing went wrong, i just run the lint test 👍🏼

**Review by jayasting98 — APPROVED** [review]
Thanks for your contribution!

**Review by cedricongjh — APPROVED** [review]
LGTM!

**Pull request merged**